.gitignore

```diff
@@ -1,6 +1,7 @@
 /node_modules
 /coverage
 /dist
+.DS_Store
```

Comment:
Can we keep this out of the repository? A [global .gitignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) is a better place for OS-specific files.